src/SFA.DAS.AANHub.Application/Apprentices/Commands/CreateApprenticeMemberCommandResponse.cs

```diff
@@ -0,0 +1,16 @@
+﻿using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.Apprentices.Commands
+{
+    public class CreateApprenticeMemberCommandResponse
+    {
+        public Guid MemberId { get; set; }
+        public string? Status { get; set; }
+
+        public static implicit operator CreateApprenticeMemberCommandResponse(Member member) => new CreateApprenticeMemberCommandResponse()
```

Comment:
add coverage for implicit operator

Reply:
Added Test [here](https://github.com/SkillsFundingAgency/das-aan-hub-api/commit/27896da38874e783783dd7f45e35229895d16eb1#diff-3815b3618bb45e02dab464e9cce56e5252465bf296d6258e0f37d5efb46bc567)